querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx

```diff
@@ -446,40 +446,93 @@ class DataDocQueryCellComponent extends React.PureComponent<IProps, IState> {
     }
 
     @bind
-    public async onRunButtonClick() {
+    public async executeQuery(options: {
```

Comment:
are we going to support the review request in the adhoc page as well?  or we can have some pointer to let user to use datadocs if they see those errors.

Reply:
added for adhoc page as well